code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -356,9 +356,9 @@
 		var/datum/reagent/the_reagent = r
 		if(istype(the_reagent, /datum/reagent/medicine))
 			medibonus += 1
-	if(creation_purity > 1) //Perfectly pure multivers gives a bonus of 2!
+	if(creation_purity >= 1) //Perfectly pure multivers gives a bonus of 2!
 		medibonus += 1
-	M.adjustToxLoss(-0.5 * min(medibonus, 3) * REM * delta_time) //not great at healing but if you have nothing else it will work
+	M.adjustToxLoss(-0.5 * min(medibonus, 3 * normalise_creation_purity()) * REM * delta_time) //not great at healing but if you have nothing else it will work
```

Comment:
since roundstart multiver starts at 0.75 purity, does this effectively nerf the tox healing of roundstart multiver?

Reply:
No - normalise_creation_purity will always return a 1 for the default reagent purity - so in this case a purity of 0.75 will be 3 and a purity of 1 will be 3.99 (maybe I should round that). So for a pure multiver adding 6 additional medicines to the patient will heal toxin at 4 per tick, whereas default will cap out at 3 from 5 medicines.